cs3api4lab/api/cs3_file_api.py

```diff
@@ -125,22 +127,20 @@ def write_file(self, file_path, content, endpoint=None, format=None):
         Write a file using the given userid as access token. The entire content is written
         and any pre-existing file is deleted (or moved to the previous version if supported).
         """
-        file_path = self.lock_manager.resolve_file_path(file_path, endpoint)
-
+        # todo fix me
+        #file_path = self.lock_api.resolve_file_path(file_path, endpoint)
```

Comment:
Does this mean that _for now_ we're not creating conflict files?

Reply:
Correct, creating conflict versions will be addressed separately, currently they are flawed and need some work